Simplenote/src/main/java/com/automattic/simplenote/NoteEditorFragment.java

```diff
@@ -801,6 +804,22 @@ public boolean onOptionsItemSelected(@NonNull MenuItem item) {
         }
     }
 
+    private void navigateToCollaborators() {
+        if (getActivity() == null || mNote == null) {
+            return;
+        }
+
+        Intent intent = new Intent(requireActivity(), CollaboratorsActivity.class);
+        intent.putExtra(CollaboratorsActivity.NOTE_ID_ARG, mNote.getSimperiumKey());
+        startActivity(intent);
+
+        AnalyticsTracker.track(
+                AnalyticsTracker.Stat.EDITOR_COLLABORATORS_ACCESSED,
+                CATEGORY_NOTE,
```

Comment:
I will not make a change about this because the file is too inconsistent with the format. In some cases, the enum type is imported completely, in others not. I will change this when I prefer to rewrite this when we change the file to Kotlin.